taxcalc/records.py

```diff
@@ -240,45 +237,18 @@ def __init__(self,
                  weights=weights_path,
                  start_year=None,
                  **kwargs):
-
-        self.read(data)
-        self.read_blowup(blowup_factors)
-        self.read_weights(weights)
-        if (start_year):
+        """
+        Records class constructor
+        """
+        self._read_data(data)
+        self._read_blowup(blowup_factors)
+        self._read_weights(weights)
+        if start_year:
             self._current_year = start_year
```

Comment:
Just as this PR mentioned, it doesn't really make any sense if we run any data other than 2009. Then it means if users random put some other start_year here, for example 2013, the calculator will just automatically skipped the blow-up process without any warning message. So my concerns are 1) what are the cases where users might put in random years here? Are those cases necessary? 2) if we want to keep this start_year attribute, should we add a message letting users know no blow-up is involved due to the start_year set up?